R/dependencies.R

```diff
@@ -191,9 +191,8 @@ dependencies <- function(
 renv_dependencies_impl <- function(
   path = getwd(),
   root = NULL,
-  ...,
```

Comment:
Can we keep `...` to force following arguments to be named?

Reply:
You may be over-estimating my discipline 😅

Reply:
Ok, I'll replace with `renv_dots_check`.